Automattic/sensei
# Update supported WP version to 6.3

## Proposed Changes

This PR updates the supported WP version to 6.3 and the minimum required version to 6.1.

## Testing Instructions

1. Install WP 6.3.
2. Make sure the plugin is working.

## Pre-Merge Checklist
